fem/fe_coll.hpp

```diff
@@ -819,6 +819,20 @@ class NURBS_HDivFECollection : public NURBSFECollection
    virtual ~NURBS_HDivFECollection();
 };
 
+/// Arbitrary order H(div) U H1  NURBS finite elements.
```

Comment:
For comments with multiple lines, `/** @brief */` is normally used.

Reply:
Thanks for the pointer, switch to `@brief`